static/src/javascripts/components/videojs-playlist-audio/videojs.playlist.js

```diff
@@ -3,10 +3,10 @@
  videojs.plugin('playlist', function(options) {
   //this.L="vjs_common_one";
   
-  var id=this.id();
- // if(typeof this.L!="undefined") var id=this.L;
-  //else workData.myPlayer.id=this.tag.id;
-  //else var id=this.id_;
+
+  console.log(this);
```

Comment:
No, but it's not our code. I don't want to modify it because it's coming in through Bower